src/app/workspace/resource/resource-instance-form/resource-instance-form.component.html

```diff
@@ -47,8 +47,10 @@
         </span>
         <span class="fill-remaining-space"></span>
         <span>
+            <!-- check each 'selected' property in addition to the forms validity because the form is actually valid when fetching ontologies and resource classes  -->
             <button mat-raised-button type="button" color="primary"
-                [disabled]="!selectResourceForm.valid || this.errorMessage" (click)="nextStep()" class="form-next">
+                [disabled]="(!selectedProject || !selectedOntology || !selectedResourceClass || !selectResourceForm.valid) || this.errorMessage" (click)="nextStep()" class="form-next">
+                <app-progress-indicator *ngIf="loading" [status]="0" class="next-progress"></app-progress-indicator>
```

Comment:
Can you set the color of the progress-indicator to primary? Just add `[color]="'primary'" `

Reply:
done in [49ddae3](https://github.com/dasch-swiss/dsp-app/pull/713/commits/49ddae3bb591887ff56610b61b40ea6e3ec932f2)